remote-config/build.gradle

```diff
@@ -17,6 +18,13 @@ android {
   defaultConfig {
     consumerProguardFiles 'proguard-rules.pro'
   }
+
+  buildTypes {
+    release {
+      useProguard true
```

Comment:
Why is this needed? In my understanding, this setting is only used during the build of the AAR and won't affect the end user. Is that wrong?

The `consumerProguardFiles` option is used for setting the proguard file which is bundled into the AAR.

Reply:
Removed. This is unnecessary.